python/packages/autogen-agentchat/src/autogen_agentchat/teams/_group_chat/_chat_agent_container.py

```diff
@@ -31,10 +33,11 @@ class ChatAgentContainer(SequentialRoutedAgent):
         agent (ChatAgent): The agent to delegate message handling to.
         message_factory (MessageFactory): The message factory to use for
             creating messages from JSON data.
+        sender_filter (List[AgentId] | None): A list of agent IDs to filter messages by sender.
```

Comment:
We don't need to use filter. Instead, we pass the manager agent's topic type to `parent_topic_type`, so participant's response sent to the manager agent. 

Then, we set up the manager agent to act as the router for messages, based on the graph definition.